SeaPkg/Library/BasePeCoffLibNegative/BasePeCoffLibNegative.c

```diff
@@ -887,6 +887,14 @@ PeCoffImageDiffValidation (
       return EFI_COMPROMISED_DATA;
     }
 
+    DEBUG ((
+      DEBUG_INFO,
+      "%a: Evaluating symbol at offset=0x%x in the target image with rule type=0x%x\n",
+      __func__,
+      ImageValidationEntryHdr->ValidationType,
+      ImageValidationEntryHdr->Offset,
+    ));

```

Comment:
Is this what was meant?

```suggestion
    DEBUG ((
      DEBUG_INFO,
      "%a: Evaluating symbol at offset=0x%x in the target image with rule type=0x%x\n",
      __func__,
      ImageValidationEntryHdr->Offset,
      ImageValidationEntryHdr->ValidationType
    ));
```

Reply:
whoops. Thanks.